.eslintrc.js

```diff
@@ -37,7 +37,14 @@ module.exports = {
         ]
       }
     ],
-    "@typescript-eslint/no-var-requires": "error",
+    "@typescript-eslint/no-require-imports": "error",
```

Comment:
Line 40 is only because `@typescript-eslint/no-var-requires` is deprecated in favor of `@typescript-eslint/no-require-imports`. See [here](https://typescript-eslint.io/rules/no-var-requires/). I didn't catch this when I originally untangled mixed use of `import` and `require` statements in the tests in [2002](https://github.com/Esri/hub.js/pull/2002)